defaults/test_default.yaml

```diff
@@ -61,6 +61,8 @@ backtrace_decoding: true
 update_db_packages: ''
 
 logs_transport: "rsyslog"
+rsyslog_imjournal_rate_limit_interval: 60  # max. 65535 till rsyslog v8.34
+rsyslog_imjournal_rate_limit_burst: 50000  # max. 65535 till rsyslog v8.34
```

Comment:
How did you calculate the "50000" value? Cannot we set maximum 65k?

Reply:
Same question: why it should be 65k? We just want to have big enough limit, but not disable rate limit at all.

But 50000 messages in 60 seconds is 25 times more than 20000 messages in 600 seconds. From the log in the original task we missed about 20k messages.

Reply:
So, if I understand your answer correctly then both values, 50k and 65k, are both too huge to care about specific values.

Reply:
Yep. We don't have a data to make more accurate calculations, and this is the reason why we make it tunable for specific jobs.